utils/docker/Dockerfile.centos.8

```diff
@@ -33,149 +33,88 @@ RUN yum clean all && yum -y install epel-release dnf-plugins-core &&	\
-      echo -e "[${MY_NAME}]\n\
-name=created from ${MY_REPO}\n\
-baseurl=${MY_REPO}\n\
-enabled=1\n\
-gpgcheck=False\n" >> /etc/yum.repos.d/local-centos-group.repo;                \
-      yum --assumeyes install yum-utils;                                      \
-      yum-config-manager --assumeyes --quiet                                  \
-                         --disable base epel extra updates;                   \
-    fi
-RUN if $QUICKBUILD; then                                                      \
-        MY_REPO="${REPO_URL}${REPO_EL8}";                                     \
-        MY_NAME="${MY_REPO#*//}";                                             \
-        MY_NAME="${MY_NAME//\//_}";                                           \
-        echo -e "[${MY_NAME}]\n\
-name=created from ${MY_REPO}\n\
-baseurl=${MY_REPO}\n\
-enabled=1\n\
-gpgcheck=False\n" >> /etc/yum.repos.d/$repo:$branch:$build_number.repo;       \
-        for repo in $REPOS; do                                                \
-            branch="master";                                                  \
-            build_number="lastSuccessfulBuild";                               \
-            if [[ $repo = *@* ]]; then                                        \
-                branch="${repo#*@}";                                          \
-                repo="${repo%@*}";                                            \
-                if [[ $branch = *:* ]]; then                                  \
-                    build_number="${branch#*:}";                              \
-                    branch="${branch%:*}";                                    \
-                fi;                                                           \
-            fi;                                                               \
-            echo -e "[$repo:$branch:$build_number]\n\
-name=$repo:$branch:$build_number\n\
-baseurl=${JENKINS_URL}job/daos-stack/job/$repo/job/$branch/$build_number/artifact/artifacts/centos7/\n\
-enabled=1\n\
-gpgcheck=False\n" >> /etc/yum.repos.d/$repo:$branch:$build_number.repo;       \
-        done;                                                                 \
-        echo "Installing: $QUICKBUILD_DEPS";                                  \
-        echo "$QUICKBUILD_DEPS" | tr '\n' '\0' | xargs -0 yum -y install;     \
-    fi
-
-ARG BULLSEYE=""
-RUN if [ "x$BULLSEYE" != "x" ]; then \
-      curl ${JENKINS_URL}job/daos-stack/job/tools/job/master/lastSuccessfulBuild/artifact/bullseyecoverage-linux.tar \
-        --retry 10 --retry-max-time 60 --silent --show-error -o bullseye.tar; \
-      mkdir -p bullseye; \
-      tar -C bullseye --strip-components=1 -xf bullseye.tar; \
-      pushd bullseye; \
-        ./install --quiet --key "${BULLSEYE}" --prefix /opt/BullseyeCoverage; \
-      popd; \
-      rm -f bullseye.tar; \
-      rm -rf bullseye; \
-    fi
-
-# set NOBUILD to disable build
```

Comment:
Why was this big hunk removed?  In particular, Quick-build.

Reply:
This did go in a commit comment but maybe it got lost.  We don't actually use Dockerfile.centos.8 in Jenkins anywhere, and it looked like the file had been copied from centos.7 and then diverged over time.  Given that we haven't run any of this code since it was copied I thought that when we do come to use it then it would be better to start off with a fresh copy and develop from there rather than a snapshot from the past that hadn't received any updates.

Reply:
Fair enough I suppose.  I just doubt that the Quick-build handling (in particular) will be any different than it is for EL7.